### PR TITLE
Node Version Checking Fix

### LINKED ISSUE #185: Node Version Checking Fix
Currently, v8.11.0 doesn't pass our node version testing, even though it is a newer version than 8.9.3

### PR DESCRIPTION
**Pull request by mugwump303**
This PR closes #185 

#### What does this PR do?
Fixes node.js version checking by changing array of strings to array of numbers.

### CODE REVIEW

File: bin/checkVersion.js [hunk 1]
```diff
@@ -9,6 +9,14 @@ const version = config.engines.node.match(/[0-9.]+/).toString();
 var min = minVersion.split('.');
 var ver = version.split('.');
 
+// turn elements in array from string to number
+min = min.map(function(value, index) {
```
Comment (wdoug) [inline bin/checkVersion.js:13]: These maps could be done on the same line as `min` is originally defined i.e.
```js
var min = minVersion.split('.').map(v => Number(v));
```

**Review by wdoug — APPROVED** [review]
:shipit:

File: bin/checkVersion.js (hunk 1 above)
Comment (mugwump303) [reply bin/checkVersion.js:13]: Should I keep it as is, or make the change?

Comment (wdoug) [reply bin/checkVersion.js:13]: Nah. I'd say leave it as is and go ahead and merge it @mugwump303

**Pull request merged**